jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -101,6 +101,39 @@ jobs:
        BAR: bar
 ```
 
+### Modifying `$PATH`
+
+**1.0**
+
+```YAML
+machine:
+  environment:
+    PATH: "/path/to/foo/bin:$PATH"
+```
+
+**2.0**
+
+In 2.0, interpolating variables into the environment is not yet supported.
+
+As a workaround, you have to add it to your `.bashrc` then make sure your command shell loads it. We can also make sure it is loaded automatically by changing `$BASH_ENV` like so.
+
+```YAML
+version: 2
+jobs:
+  build:
+    working_directory: /tmp
+    docker:
+      - image: buildpack-deps:jessie
+    environment:
+       BASH_ENV: ~/.bashrc
+  steps:
+    run: echo "export PATH=/path/to/foo/bin:$PATH" > ~/.bashrc
```

Comment:
actually, I would recommend changing the line to

```
echo 'export PATH=/path/to/foo/bin:$PATH' >> $BASH_ENV
```

Also, note `>>` (to avoid overwriting the file) and use of single-quotes `'` to allow for delayed expansion if that's desired.